src/index.tsx

```diff
@@ -1,22 +1,34 @@
-import { NativeModules, Platform } from 'react-native';
-
-const LINKING_ERROR =
-  `The package '@iterable/react-native-sdk' doesn't seem to be linked. Make sure: \n\n` +
-  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
-  '- You rebuilt the app after installing the package\n' +
-  '- You are not using Expo Go\n';
-
-const ReactNativeSdk = NativeModules.ReactNativeSdk
-  ? NativeModules.ReactNativeSdk
-  : new Proxy(
-      {},
-      {
-        get() {
-          throw new Error(LINKING_ERROR);
-        },
-      }
-    );
-
-export function multiply(a: number, b: number): Promise<number> {
-  return ReactNativeSdk.multiply(a, b);
-}
+/**
```

Comment:
This seems like a weird replacement.

Reply:
This came with the scaffolding as an example function